unittests/tools/test_checkmarx_one_parser.py

```diff
@@ -125,7 +128,7 @@ def test_sca_finding(finding):
             findings = parser.get_findings(testfile, Test())
             self.assertEqual(146, len(findings))
             for index in range(len(findings)):
-                print(index, findings[index])
+                logger.debug(index, findings[index])
```

Comment:
Found one more 😉 
```suggestion
                logger.debug(f"{index} {findings[index]}")
```

Reply:
Looks like this one might have been reverted in a rebase?

Reply:
@cneill, you are probably right. Thanks also for other suggestions.